src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -288,7 +290,7 @@
      (fn []
        ~@body)))
 
-(defn with-query-results-cursor*
+(defn with-query-results-cursor
```

Comment:
Ah wasn't sure about that, yeah how about I make a ticket to rename it and refactor the code from your other comment, that way the refactor will save some renaming hell?